locale/en/blog/npm/managing-node-js-dependencies-with-shrinkwrap.md

```diff
@@ -16,6 +16,8 @@ layout: blog-post.hbs
 Photo by Luc Viatour (flickr)
 </p>
 
+**This post is outdated:** Please refer to official npm documentation on [shrinkwrap](https://docs.npmjs.com/cli/v8/configuring-npm/npm-shrinkwrap-json) and/or [lockfiles](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json) going forward.
```

Comment:
Optional suggestion:

```suggestion
**This post is outdated.** Please refer to official npm documentation on [shrinkwrap](https://docs.npmjs.com/cli/v8/configuring-npm/npm-shrinkwrap-json) and/or [lockfiles](https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json) instead.
```

Another totally optional suggestion: Maybe hide the post content in a `<details>` tag that says "Original content" or something like that so people have to take an extra step to see it.